src/home/room_screen.rs

```diff
@@ -2261,9 +2261,10 @@ impl RoomScreen {
         //   because the location might change by the next time the user opens this same room.
         self.location_preview(id!(location_preview)).clear();
         submit_async_request(MatrixRequest::SubscribeToTypingNotices {
-            room_id,
+            room_id: room_id.clone(),
             subscribe: false,
         });
+        submit_async_request(MatrixRequest::SubscribeToOwnUserReadReceiptsChanged { room_id, subscribe: false });
```

Comment:
follow above formatting:
```suggestion
        submit_async_request(MatrixRequest::SubscribeToOwnUserReadReceiptsChanged {
            room_id,
            subscribe: false,
        });
```